pkgs/by-name/fg/fgqcanvas/package.nix

```
@@ -0,0 +1,51 @@
+{ fetchgit
+, pkg-config
+, stdenv
+, lib
+# Package dependencies
+, qt5
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fgqcanvas";
+  version = "unstable-11-02-2024";
```

```suggestion
  version = "unstable-2024-02-11";
```
YYYY-MM-DD is the internet standard -> RFC3339, ISO8601

---

Technically the current convention is `[last_upstream_tag]-unstable-YYYY-MM-DD` (ref: <https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming>), but admittedly a lot of packages still don't follow that convention and it's not very discoverable